index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/main.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <main>
+          <div id="top-social-media">
+            <span class="icon"><img src="./Images/Icons/facebook_f.svg" alt=""></span>
+            <span class="icon"><img src="./Images/Icons/twitter_squared.svg" alt=""></span>
+            <p>English</p>
+            <p>My Page</p>
+            <p>Logout</p>
+          </div>
+      <header>
+        <div id="drop-menu">
+          <img src="Images/Icons/humberge.svg" alt="show menu" />
+        </div>
+        <div id="logo">
+          <a href="./"><img src="./Images/Icons/logo.svg" alt="" /></a>
+        </div>
+        <nav class="hide">
+          <div id="close-menu">
+            <img src="./Images/Icons/delete.svg" alt="close menu" />
+          </div>
+          <ul>
+              <li>
+                <a href="./">Home</a>
+              </li>
+            <li>
+              <a href="about.html">About</a>
+            </li>
+            <li>
+              <a href="#main-tutors">Classes</a>
+            </li>
+            <li>
+              <a href="#customers">Reputation</a>
+            </li>
+            <li>
+              <a href="#partners">Sponsor</a>
+            </li>
+            <li>
+              <a href="#">News</a>
+            </li>
+            <li>
+              <a href="#">WM campaign</a>
+            </li>
+          </ul>
+          <div class="remaining"></div>
+        </nav>
+      </header>
+      <section id="home">
+        <p class="title-3">"Hello! Website developers"</p>
+        <h1 class="title-1">
+          <img src="Images/pictures/Main-text.svg" alt="" />
+        </h1>
+        <div class="card">
+          <small>
+            Our tutorials and content help students learn the ins and outs of
+            Web development. These tutorials are tailored to students of
+            different levels.
+          </small>
+        </div>
+        <h1 class="title-1">2022.3.7(Mon) ~ (Today)</h1>
```

Comment:
Please be sure to only have one `h1` tag in your HTML to meet with semantics and also for SEO reasons 🔥

Reply:
@mateo951 Nice view. ill correct it right away.